packages/core/src/cart/headless-cart/headless-cart.ts

```diff
@@ -0,0 +1,141 @@
+interface BaseFieldFragment {
+    value: number;
+}
+
+export interface HeadlessLineItem {
```

Comment:
Is it a headlessCart or a GQL cart response?

Reply:
This is GraphQL cart response

Reply:
Can we rename it accordingly, since headless cart doesn't necessarily mean GQL. Something like `CartQueryResponse` or `GQLCart`.

Reply:
@animesh1987 In this case it will be like `CartQueryLineItem`. Are you ok with that?

Reply:
In this case i have to rename all files including interfaces and folder

Reply:
Yes please.